scripts/storage-proxy-spawn-ceph-clusters/install_storage_proxy.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+
+sudo chmod 755 /
+cd $HOME/
+sudo apt update
+sudo apt-get install -y build-essential git-core libreadline-dev libsqlite3-dev libssl-dev libbz2-dev tk-dev libzmq3-dev libsnappy-dev libffi-dev zlib1g-dev wget curl llvm libncursesw5-dev xz-utils  libxml2-dev libxmlsec1-dev  liblzma-dev
+
+curl -fsSL https://get.docker.com -o get-docker.sh
+sudo sh get-docker.sh
+sudo apt-get install docker-compose
+sudo addgroup docker
+sudo adduser vagrant docker
+
+sudo apt-get install git-lfs
+git lfs install
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+source "$HOME/.cargo/env"
+
+curl https://pyenv.run | bash
+export PYENV_ROOT="$HOME/.pyenv"
+command -v pyenv >/dev/null || export PATH="$PYENV_ROOT/bin:$PATH"
+eval "$(pyenv init -)"
+eval "$(pyenv virtualenv-init -)"
+
+echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+echo 'eval "$(pyenv init -)"' >> ~/.profile
+
+echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.bashrc
+echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.bashrc
+echo 'eval "$(pyenv init -)"' >> ~/.bashrc
+
+echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.profile
+echo 'eval "$(pyenv virtualenv-init -)"' >> ~/.bashrc
```

Comment:
Instead of doing implicitly, how do you think about letting user to install pyenv by themselves?

Reply:
Yes. Users can install by themselves. This is done for the convenience of a developer who wants to spawn ceph for testing purposes only via Vagrant VM to simulate multiple nodes.  
However, I don't think the installation scripts spawn-ceph-cluster should be included in a final pull request.